src/content/configuration/stats.md

```diff
@@ -15,6 +15,8 @@ The `stats` option lets you precisely control what bundle information gets displ
 
 T> For webpack-dev-server, this property needs to be in the `devServer` object.
 
+T> For webpack-dev-middleware, this property needs to be in the `webpack-dev-middleware options` object.
```

Comment:
What is exactly the name of the option?
`webpack-dev-middleware options`?

Reply:
Yes this PR is correct, but formatting is wrong. go with this property needs to be:
in the webpack-dev-middleware's `options` object.

You can find the example here for the reference: https://github.com/webpack/webpack-dev-middleware#usage